Game/Scripts/BixAttackScript.cpp

```diff
@@ -243,6 +212,72 @@ void BixAttackScript::HeavyFinisher()
 	}
 }
 
+void BixAttackScript::JumpFinisher()
+{
+	animation->SetParameter("IsJumpAttacking", true);
+	isAttacking = true;
+
+	jumpFinisherScript->PerformGroundSmash(15.0f, 4.0f); // Bix jumping finisher
+	//jumpFinisherScript->ShootForceBullet(15.0f, 4.0f); // Allura jumping finisher, placed it here for now
+
+	comboSystem->SuccessfulAttack(-35, AttackType::HEAVYFINISHER);
+}
+
+void BixAttackScript::ResetAttackAnimations()
+{
+	switch (currentAttack)
+	{
+		case AttackType::LIGHTNORMAL:
+			if (!animation->isPlaying())
+			{
+				animation->SetParameter("IsAttacking", false);
+				isAttacking = false;
+			}
+			break;	
+
+		case AttackType::HEAVYNORMAL:
+			if (!animation->isPlaying())
+			{
+				animation->SetParameter("IsAttacking", false);
+				isAttacking = false;
+			}
+			break;	
```

Comment:
Since these two cases are the same, we could merge them.
```suggestion
		case AttackType::LIGHTNORMAL:
		case AttackType::HEAVYNORMAL:
			if (!animation->isPlaying())
			{
				animation->SetParameter("IsAttacking", false);
				isAttacking = false;
			}
			break;	
```

Reply:
They are now, but they won't be.

Reply:
They aren't anymore.